etc/system_config.json

```diff
@@ -69,8 +122,9 @@
             "init": {
                 // This impl allows relative paths, to be interpreted relative
                 // to system DATA_DIR / WORK_DIR respectively.
-                "model_directory": "ContentDescriptors/ColorDescriptor/csift/example_image",
-                "work_directory": "ContentDescriptors/ColorDescriptor/csift/example_image",
+//                "descriptor_type_str": "csift",
```

Comment:
Style, but inconsistent comment indentation. Also, same comment above as to adding description for parameters that are commented out.